blog/2020-05-08-AIVersusCovid-19.adoc

```diff
@@ -0,0 +1,135 @@
+change to the ward "COVID-19 Ward 1". Every shift has a small circle at the top of
+it denoting the Covid-19 susceptibility of its employee. A shield represents
+inoculated employees; people who have acquired immunity to Covid-19. A near empty
+circle mean the employee has a low Covid-19 susceptibility, while a more filled in
+circle mean a higher Covid-19 susceptibility. Finally, a red circle mean the employee
+is extremely susceptible to Covid-19, and should not placed in a ward with Covid-19.
+Notice how there are no Extreme risk employees in the Covid-19 ward? Moreover, most
+of the Covid-19's ward shifts either have inoculated or low risk employees.
+
+image::CovidWardNoExtremeRisk.png[No Extreme Risk Employees in Covid-19 Ward]
+
+One second, I am getting a call. Okay, I'm back. Beth P. Watt caught the flu and
+as such is not able to make their shift this Saturday. Luckily for us, OptaPlanner
+can handle sudden changes well. Go to the "Availability" tab. Click the Employee
+selector (left side of the toolbar) and type in "Beth P. Watt". Click the Clock
+icon on the week selector to jump to the current week. Now click the area under
+the Saturday label on the calendar. This creates a new unavailability for the
+employee Beth P. Watt on Saturday.
+
+image::EmployeeUnavailable.png[Beth P. Watt is unavailable]
+
+You should now see a message indicating the schedule is not feasible in the
+toolbar. This is because Beth P. Watt is unavailable on Saturday but is
+assigned to a shift on Saturday. Let's fix that. Navigate to the "Adjustments"
+tab, and click "Reschedule". You should notice OptaPlanner quickly fixed the
+schedule by assigning Beth P. Watt's two shifts to another employee. Also, note
+that OptaPlanner changed only those two shifts; OptaPlanner will try to minimize
+the number of published shifts changed (after all, nurses and doctors have a
+life, which they plan around their schedules).
+
+image::ScheduleAfterReplanning.png[Only the two shifts Beth P. Watt was unavailable for changed]
+
+A few other things you can do with this application:
+
+* You can "pin" an employee to a particular shift, meaning OptaPlanner
+will not change that shift's employee when solving.
+
+* You can create a rotation for the employees, giving them more consistency
+in their day to day lives and allowing automatic provisioning of new shifts.
+
+== Conclusion
+
+As you just saw, OptaWeb Employee Rostering allows you to easily create
+and optimize rosters, that does not only minimize Covid-19's effect on
+health care workers, but also maximizes their happiness. This was only
+possible as OptaPlanner has such a flexible and easy way to add new constraints,
+allowing us to quickly modify OptaWeb Employee Rostering to accommodate
+the new constraints that came with Covid-19. You might also be interested
+in https://developers.redhat.com/devnation/tech-talks/AI-vs-covid19-java/[Geoffery De Smet's talk on the application].
```

Comment:
Also add a link to the source on github (correct branch) in the conclusion, togher with something "follow the readme to build and run it locally"

Reply:
I put the repo link in the "Using the application" section, since I am basically giving users directions on how to use the application, which only make sense if they could build/run it first. I might add it again in the conclusion.